bedshift/bedshift.py

```diff
@@ -434,7 +492,58 @@ def read_bed(self, bedfile_path, delimiter='\t'):
         df[3] = 0 # column indicating which modifications were made
         return df
 
+def _read_from_yaml(fp):
+    """
+    Loads yaml config data
 
+    :param float fp: the path to the configuration file for multiple add_from_file and drop_from_file
+    :return int: loaded yaml data
+    """
+    with open(fp, "r") as yaml_file:
+        config_data = yaml.load(yaml_file, Loader=yaml.FullLoader)
+    return config_data
+
+def handle_yaml(fp):
+    bedshifter = Bedshift(fp)
+    data = _read_from_yaml(fp)
+    operations = [operation for operation in data["bedshift_operations"]]
+
+    for operation in operations:
```

Comment:
We should try to consider the cases where the same operation can be specified multiple times in the same yaml. For example, user wants
```
add_from_file:
- file: xyz.bed
- rate: 0.2
add_from_file:
- file: "abc.bed"
- rate: 0.5
add_from_file:
- file: "abc.bed"
- rate: 0.33
 ```

Reply:
it can already do that.

Reply:
(with the revised version now)